.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: File a report about a problem with the Operator
+title: ''
+labels: ''
+assignees: ''
+
+---
+**What did you do to encounter the bug?**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**What did you expect?**
+A clear and concise description of what you expected to happen.
+
+**What happened instead?**
+A clear and concise description of what happened instead
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Operator Information**
+ - Operator Version
+ - MongoDB Image used
+
+**Kubernetes Cluster Information**
+ - Distribution:
+ - Version:
+ - Image Registry location (quay, or an internal registry)
```

Comment:
[nit] `Distribution` and `Version` ends with a colon `:`, Image Registry location does not. Maybe we should add it there too?

Reply:
yep!